packages/core/DEVELOPMENT.md

```diff
@@ -1,92 +1,40 @@
 # Development
 
-## Development environment
+### Prerequisites
 
-> Helsinki Design System uses [**Lerna**](https://lerna.js.org/) for running scripts across the repo as well as versioning and creating releases of the packages. [**Yarn workspaces**](https://yarnpkg.com/lang/en/docs/workspaces/) is used to manage dependencies. This allows the separate packages to reference each other via symlinks during local development.
+Set up your local development environment by following the steps in the [development guide](../../DEVELOPMENT.md#setting-up-local-development-environment)
```

Comment:
You're missing a dot from the end of the link text here, it should be `[development guide.]`